AngleSharp.Scripting.CSharp/AngleSharp.Scripting.CSharp.csproj

```diff
@@ -43,8 +43,9 @@
     <Compile Include="WindowService.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="AngleSharp">
+    <Reference Include="AngleSharp, Version=0.8.7.42712, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
No problem. I actually don't know why the package restore did not work; I think there are no specific settings for package restore since VS does that automatically since ~2013 or so. But still there are some hiccups from time to time.